projects/ngx-d3timeline/src/lib/content/event-rectangle.component.spec.ts

```diff
@@ -22,6 +22,7 @@ describe('EventRectangleComponent', () => {
 
   it('should render correctly', () => {
     fixture.componentInstance.eventRectangle = {
+      id: 1,
```

Comment:
does this change the snapshot?

Reply:
No the id doesn't get rendered in the html